migrations/2_deploy.js

```diff
@@ -4,6 +4,7 @@ const PoolDeployer = require('./utils/PoolDeployer');
 const initSettings = require('./utils/init_settings');
 
 const ERC20 = artifacts.require("@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol");
+const AdminUpgradeabilityProxy = artifacts.require("./base/UpgradeableProxy.sol");
```

Comment:
We could log a message including AdminUpgradeabilityProxy's address

Reply:
We won't be deploying this address on its own. It will only be deployed when when want to create a brand new upgradable contract. Also for the ones we are creating they are being logged via the DeployerApp [here](https://github.com/teller-protocol/teller-protocol-v1/pull/118/files/6ce23521c0e7a4ab608687abc9d5fd504ff0ca74#diff-0c3e3d003115713a42f37b1f3e04c46fR18-R19)